week-4/InClass/A-objects-intro/exercise-part-1.js

```diff
@@ -7,3 +7,43 @@ Assign each of them to a separate variable
 
 */
 
+let car = {
+  make: "VW",
+  model: "passat",
+  color: "blue",
+  gear: "auto",
+  seat: 5,
+  year: 2007,
+  hp: 200,
+  price: 2000,
+  engineSize: 2.0,
+  fuel: "petrol",
+  cylinder: 4,
+  acceleration: "0-100 @5 sec"
+};
+let bottle = {
+  contain: "water",
+  volume: 2,
+  height: 25,
+  colour: "transparent",
+  price: 17
+};
+let iron = {
+  brand: "tefal",
+  model: "steam",
+  color: "black",
+  voltage: 1700
+};
+let bicycle = {
+  color: "black",
+  model: "electric",
+  brand: "iRider",
+  madeIn: "china",
+  maxSpeed: 30
+};
+let book = {
+  name: "Bible",
+  pages: 500,
+  coverColour: "black",
+  language: "persian"

```

Comment:
I thought "Aramaic"?

Reply:
"Aramic" is a part of Persian language which was the main language of Iran in the `Sasanian Empire` Era.